components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+﻿import {apiTryLogin} from "../../api/loginApiClient.module";
+import React, {FormEvent, useState} from "react";
+import scss from "./LoginForm.module.scss";
+
+interface LoginFormProps {
+    userId: string;
+    password: string;
+    setUserId: Function;
+    setPassword: Function;
```

Comment:
You can make TS' restrictions tighter by specifying the parameters and return type of these functions: `(userId: string) => void`.

Reply:
I did wonder how to do this